proto-docs/status.md

```diff
@@ -0,0 +1,138 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [status/types.proto](#status/types.proto)
+
+  - Messages
+    - [Status](#neo.fs.v2.status.Status)
+    - [Status.Detail](#neo.fs.v2.status.Status.Detail)
+    
+
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="status/types.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## status/types.proto
+
+
+ <!-- end services -->
+
+
+<a name="neo.fs.v2.status.Status"></a>
+
+### Message Status
+Declares the general format of the status returns of the NeoFS RPC protocol.
+Status is present in all response messages. Each RPC of NeoFS protocol
+describes the possible outcomes and details of the operation.
+
+Enumerated with code. Any unique result of an operation in NeoFS is
+unambiguously associated with the code value.
+
+To facilitate the description of codes, enumerations are defined that
+are one-to-one corresponding to sections. Values can be referred to in
+the following ways:
+
+* numerical value ranging from 0 to 4,294,967,295 (global code);
+
+* values from enumeration (local code). The formula for the ratio of the
+  local code (`L`) of a defined section (`S`) to the global one (`G`):
+  `G = 1024 * S + L`.
+
+All outcomes are divided into successful and failed, which corresponds
+to the success or failure of the operation. The definition of success
+follows from the semantics of RPC and the description of its purpose.
+The server must not attach code that is the opposite of outcome type.
+
+See the set of return codes in the description for calls.
+
+Each status can carry developer-facing error message. It should human
+readable text in English. The server should not transmit (and the client
+should not expect) useful information in the message. Field `details`
+should be used to detail the return.
```

Comment:
...should make the return more detailed.

Reply:
Changed.